### PR TITLE
add documentation on the airbyte-ci development process

### PR DESCRIPTION
**Pull request by stephane-airbyte**

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
* `master`
  * **#34543** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/34543?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stephane-airbyte and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-ci/connectors/pipelines/README.md [hunk 1]
```diff
@@ -91,6 +91,24 @@ However, this will not automatically install the dependencies for the local depe
...
+From `airbyte-ci/connectors/pipelines`:
+```bash
+poetry run pytest tests
+```
+
+You can also run a subset of tests:
+```bash
+poetry run pytest pipelines/models/steps.py
+```
+
```
Comment (erohmensing) [inline airbyte-ci/connectors/pipelines/README.md:105]:
```suggestion

More options, such as running test by keyword matching, are available - see the [pytest CLI documentation](https://docs.pytest.org/en/6.2.x/usage.html) for all the available options.```

**Review by erohmensing — APPROVED** [review]
Thanks for adding this! Added a suggestion for more available options, but solid addition 👍🏻

**Comment by stephane-airbyte** [comment]
### Merge activity

* **Jan 27, 5:17 PM**: @stephane-airbyte merged this pull request with [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/34543).

**Pull request merged**